### PR TITLE
Add jarvis-tools recipe

### PR DESCRIPTION
**Pull request by wd15**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/jarvis-tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - jarvis.ai
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://jarvis.nist.gov
+  summary: 'jarvis-tools: an open-source software package for data-driven atomistic materials design'
+  license: NIST-PD
```
Comment (saraedum) [inline recipes/jarvis-tools/meta.yaml:46]: I am a bit confused about the license here. The LICENSE file you included is not identical to https://spdx.org/licenses/NIST-PD.html (the one you included essentially has that extra paragraph at the end.)

File: recipes/jarvis-tools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  license_file: LICENSE
```
Comment (saraedum) [inline recipes/jarvis-tools/meta.yaml:47]: Did you by any chance create an upstream issue that the license should be packaged in their tarball?

File: recipes/jarvis-tools/LICENSE [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Terms of Use
```
Comment (saraedum) [inline recipes/jarvis-tools/LICENSE:1]: This does not seem to be the exact license file that is in the upstream repository: https://github.com/usnistgov/jarvis/blob/master/LICENSE

Is that intentional?

File: recipes/jarvis-tools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - joblib
+    - matplotlib-base
+    - numpy
+    - pytest
```
Comment (saraedum) [inline recipes/jarvis-tools/meta.yaml:26]: That's a surprising dependency in `run`. I see that upstream has this in their setup.py as well. Is this really needed?

File: recipes/jarvis-tools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
```
Comment (saraedum) [inline recipes/jarvis-tools/meta.yaml:22]: I am a bit surprised that there are no minimum versions here at all. I see that upstream does not have any version requirements in their setup.py either. Do you think that jarvis-tools actually work with any version of these dependencies?

**Review by saraedum — COMMENTED** [review]
This looks quite reasonable to me but maybe you could clarify what's actually the license here :)

File: recipes/jarvis-tools/meta.yaml (hunk 1 above)
Comment (wd15) [reply recipes/jarvis-tools/meta.yaml:46]: Sorry, the official [Jarvis license file](https://github.com/usnistgov/jarvis/blob/master/LICENSE) is mangled. Thanks for checking it.

However, the official NIST license is [here](https://www.nist.gov/open/copyright-fair-use-and-licensing-statements-srd-data-software-and-technical-series-publications). It's close, but not exactly what's listed at [spdx.org](https://spdx.org/licenses/NIST-PD.html). I think the official location for the license is at NIST itself though.

File: recipes/jarvis-tools/meta.yaml (hunk 2 above)
Comment (wd15) [reply recipes/jarvis-tools/meta.yaml:47]: Done, https://github.com/usnistgov/jarvis/issues/128

File: recipes/jarvis-tools/LICENSE (hunk 3 above)
Comment (wd15) [reply recipes/jarvis-tools/LICENSE:1]: No, oversight. Will fix. The upstream version is also mangled.

File: recipes/jarvis-tools/meta.yaml (hunk 1 above)
Comment (wd15) [reply recipes/jarvis-tools/meta.yaml:45]: I've update to the official NIST license and add an issue to fix upstream, https://github.com/usnistgov/jarvis/issues/127

File: recipes/jarvis-tools/meta.yaml (hunk 4 above)
Comment (wd15) [reply recipes/jarvis-tools/meta.yaml]: It's been removed. Definitely not a run time dependency. Added an issue to have it removed, https://github.com/usnistgov/jarvis/issues/129

File: recipes/jarvis-tools/meta.yaml (hunk 1 above)
Comment (wd15) [reply recipes/jarvis-tools/meta.yaml]: Actually the license listed at spdx.org is totally mangled. It's wrong and needs to be fixed.

File: recipes/jarvis-tools/meta.yaml (hunk 5 above)
Comment (wd15) [reply recipes/jarvis-tools/meta.yaml:22]: Determining the minimum versions for all the packages would be painstaking. I am involved with some other projects ([fipy](https://github.com/conda-forge/fipy-feedstock/blob/master/recipe/meta.yaml) and [pymks](https://github.com/conda-forge/pymks-feedstock/blob/master/recipe/meta.yaml) and we don't do that).

File: recipes/jarvis-tools/meta.yaml (hunk 4 above)
Comment (wd15) [reply recipes/jarvis-tools/meta.yaml:26]: Scrub that. It's listed in setup.py so is needed to build I assume.

**Comment by wd15** [comment]
@saraedum, I think this is ready for you to look over again.

**Pull request merged**